src/components/views/ViewsListTable.tsx

```diff
@@ -90,33 +114,15 @@ const ViewsListTable: React.FunctionComponent = () => {
                             disableSelectionOnClick
                             hideFooter
                             onRowClick={ (row) => {
-                                router.push(`/organize/${orgId}/people/views/${row.id}`);
+                                if (row.id !== selectedViewToDelete)
+                                    router.push(`/organize/${orgId}/people/views/${row.id}`);
```

Comment:
I think there should be curly braces around all blocks, even single-line blocks.

Reply:
done